packages/providers/mgt-msal-provider/src/MsalProvider.ts

```diff
@@ -192,10 +200,10 @@ export class MsalProvider extends IProvider {
   public async login(authenticationParameters?: AuthenticationParameters): Promise<void> {
     let loginRequest: AuthenticationParameters = authenticationParameters || {
       loginHint: this._loginHint,
-      prompt: 'select_account',
       scopes: this.scopes
```

Comment:
What is the default prompt? this could be a breaking change - should we use `select_account` by default if it's not set?

Reply:
Based on the documentation, select_account sounds like our default functionality:
"sends the user to an account picker where all of the accounts remembered in the session will appear"

Reply:
We should try to keep that functionality - otherwise we are introducing a breaking change. You could set a default in [`initProvider`](https://github.com/microsoftgraph/microsoft-graph-toolkit/pull/998/files#diff-0a29bc859cd5c9b75d0becac157312f5fc9091b6143b6e240af0d63e8fce7647R395), similar to how `this._loginType` is set

Reply:
I couldn't push directly to the `next/domainHint` because its a protected branch but I committed as a suggestion, can you see if you can approve/push it?